openpower/configs/linux/skiroot_defconfig

```diff
@@ -209,7 +209,6 @@ CONFIG_IPMI_DEVICE_INTERFACE=y
 CONFIG_IPMI_POWERNV=y
 CONFIG_IPMI_WATCHDOG=y
 CONFIG_HW_RANDOM=y
-CONFIG_TCG_TPM=y
```

Comment:
Is this intentional? My understanding is that CONFIG_TCG_TIS_I2C_NUVOTON depends on CONFIG_TCG_TPM, so the next value will be masked if we make olddefconfig?

Reply:
IMA selects TCG_TPM:
```
security/integrity/ima/Kconfig:12:      select TCG_TPM if HAS_IOMEM && !UML
```

Reply:
Thanks for the clarification!